fractal-theme/assets/scss/src/_elements.scss

```diff
@@ -122,4 +122,8 @@
       }
     }
   }
+
+  h3 {

```

Comment:
I'm not even going to question this :D

Reply:
hehe

Reply:
> Also, what are your thoughts on a light variation in the rich text component?

Do you mean it's broken? If this is what you mean then this PR fixes the issue. The pull request enables a dark and light rich-text possibility :)